services/portal/GWASpp/GWASppProps.js

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+    path: 'analysis/GWASUIApp',
+    resultsPath: 'analysis/GWASResults',
+
+    // GWAS++ elements
+    NextSpan: '//span[contains(text(),\'Next\')]',
+    PreviousSpan: '//span[contains(text(),\'Previous\')]',
+    GWASWindow: '//div[@class="select-container"]',
+    checkedRadio: '(//*[@class="ant-radio ant-radio-checked"])',
+    // from step1
+    AddCohortButton: '//button[contains(text(),"Add New Cohort")]',
```

Comment:
@pieterlukasse & @nmetokishlubsky  discussed this: we're concerned that identifying and testing elements by strings will lead to brittle tests. For example, it is possible that the text might need to change from "Add New Cohort" to "Add New Cohorts" and then the tests would need to be revised. 

We suggest that we add id attributes or data-testid attributes and then the string checking to limit itself to checking for for length. Let us know what you think, this approach would likely require us to update the web application with the needed id or data-testid values, and then the test to be updated to reference those instead of checking for exact strings in a follow up PR.

Reply:
So here I am using Relative Xpath for the button, which looks for the button element with text. Yes, if the button text changes in the future, we would have to come back and change the text for the Rel Xapth. 

We can also rely on CSS selector for the button. But this will also would end up needing a change if the button changes or any other case. Let me try with the CSS selector and let me know how you feel about it @jarvisraymond-uchicago